misc/prime_seive.c

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file
+ * @brief [Prime Seive](https://leetcode.com/problems/count-primes/)
+ * algorithm implementation.
+ * @author[Divyansh Kushwaha](https://www.github.com/webdesignbydivyansh)
+ */
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+constexpr int MAX_SIZE=1000000;
```

Comment:
You can use this as a template on functions using `size_t`.
Then, on `main` function, create a variable with the name `MAX_SIZE` with the value of `1000000`.

Reply:
It is better that I write 1000000 rather than creating a new variable!
I hope the program compiles and goes well...

Reply:
It might sound it's better, but what if you wanna change the variable value?
You must go and change all sentences to the new value. Easiest thing would be to change the variable value.

Reply:
- `const int MAX_SIZE = 1000000` 
- `#define MAX_SIZE 1000000`
anyone of them will work fine

Reply:
But just a few moments ago, you told me to revert my changes as you had mislooked this program as c++ instead of c

Reply:
The methods above described by @ayaankhan98 will work fine on `.c` files.

Reply:
@webdesignbydivyansh by revert i actully means to revert the last commit of `constexpr` changes.
hope things are clear now.